Guide/realtime-spas.markdown

```diff
@@ -597,6 +597,81 @@ const todos = await query('todos').fetchAndRefresh(callback);
 
 The `fetchAndRefresh` function is using a websocket to be notified about any changes to the selected data set. [It's using IHP's `DataSubscription` API.](https://github.com/digitallyinduced/ihp/blob/master/lib/IHP/static/vendor/ihp-datasync.js) For more fine grained control you can use the `DataSubscription` API directly instead of relying on `fetchAndRefresh`.
 
+#### Filtering
+
+Filtering records works similar to normal IHP. In JavaScript you have the choice between `filterWhere` and `where` - the two function work identically.
```

Comment:
To simplify I'd decide on one version here (IMO `where` is better as it's shorter), and then not mention the alternative anywhere in the docs (maybe only mention it as a legacy version for keeping b.c.).

E.g. this sentence here

>  If you want to check for inequality instead of equality of a column value, you can use the `filterWhereNot` and `whereNot` functions. They work identically to their `filterWhere` and `where` counterparts and can be used in any situation those functions can be used in.

Could be simplified if it was just using one variant of the functions.

Reply:
Makes sense. I'll make `where` and `whereNot` the default functions, aliasing the old `filterWhere` and `filterWhereNot` functions to those (instead of the way it currently is) and update the docs :+1: